src/helpers/overrides.js

```diff
@@ -14,9 +14,14 @@ export type StyleOverrideObjectT = {};
 
 export type StyleOverrideT = StyleOverrideObjectT | StyleOverrideFunctionT;
 
+export type PropsOverrideFunctionT = ({}) => ?{};
```

Comment:
can we get a test asserting what happens when the return value of a props function is null/undefined?